extensions/devservices/common/src/main/java/io/quarkus/devservices/common/ContainerAddress.java

```diff
@@ -0,0 +1,23 @@
+package io.quarkus.devservices.common;
+
+public class ContainerAddress {
+    private final String ipAddress;
+    private final int port;
+
+    public ContainerAddress(String ipAddress, Integer port) {
+        this.ipAddress = ipAddress;
+        this.port = port;
+    }
+
+    public String getIpAddress() {
```

Comment:
What about just `getHost`.
First, it may not be an IP, but a name (like "localhost"), and from a user POV, we just need the host name (how is it resolved is irrelevant).

Reply:
Yes, this is a better name.